src/userobjects/MyTRIMRasterizer.C

```diff
@@ -356,7 +358,7 @@ MyTRIMRasterizer::finalize()
   if (_trim_parameters.desired_npka == 0 || _trim_parameters.desired_npka > _trim_parameters.original_npka)
   {
     _trim_parameters.scaled_npka = _trim_parameters.original_npka;
-    _trim_parameters.result_scaling_factor = 1;
+    _trim_parameters.result_scaling_factor = 1.0 / _trim_parameters.recoil_rate_scaling;
 
     if (_trim_parameters.desired_npka > _trim_parameters.original_npka)
       mooseDoOnce(mooseWarning("number_pka is larger than number of sampled PKAs and will be ignored!"));
```

Comment:
Not quite part of this PR, but I don;t like this warning. We should name the parameter `max_pka_count` and drop the warning.